couchbase-entity-api/src/main/java/kaufland/com/coachbasebinderapi/DocId.java

```diff
@@ -0,0 +1,14 @@
+package kaufland.com.coachbasebinderapi;
```

Comment:
Should be couchbase instead of coachbase. But that would be a bigger change, maybe separate PR.

Reply:
would be also a breaking change